llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1706,17 +1706,22 @@ class VPWidenGEPRecipe : public VPRecipeWithIRFlags {
 
 /// A recipe to compute a pointer to the last element of each part of a widened
 /// memory access for widened memory accesses of IndexedTy. Used for
-/// VPWidenMemoryRecipes that are reversed.
+/// VPWidenMemoryRecipes or VPInterleaveRecipes that are reversed.
 class VPVectorEndPointerRecipe : public VPRecipeWithIRFlags,
                                  public VPUnrollPartAccessor<2> {
   Type *IndexedTy;
 
+  /// The constant stride of the pointer computed by this recipe.
```

Comment:
```suggestion
  /// The constant stride of the pointer computed by this recipe. The stride will be scaled by IndexedTy.
```

Would be good to make clear that the stride is not scaled, but a strid ein terms of IndexedTy unless I am missing something. There might be a clearer way to express this than my suggestion above.